src/interfaces/IAxelarGasReceiver.sol

```diff
@@ -0,0 +1,102 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.9;
+
+// This should be owned by the microservice that is paying for gas.
+interface IAxelarGasReceiver {
+    error NothingReceived();
+    error InvalidCodeHash();
+    error SetupFailed();
+
+    event Upgraded(address indexed newImplementation);
+    event OwnershipTransferred(address indexed newOwner);
+
+    event GasPaidForContractCall(
+        address sourceAddress,
+        string destinationChain,
+        string destinationAddress,
+        bytes32 payloadHash,
+        address gasToken,
+        uint256 gasAmount
+    );
+    event GasPaidForContractCallWithToken(
```

Comment:
I know it's a nitpick, however there are inconsistencies between new lines between blocks of code. In my opinion, every block of code should be separated by new lines, whether they are multi line event declarations, if blocks, event emissions, etc etc.